ckanext/dhis2harvester/templates/source/pivot_table_refresh.html

```diff
@@ -8,7 +8,7 @@
 <form id="source-new" class="dhis2-source-new form-horizontal dataset-form {{ h.bootstrap_version() }}" method="post" >
   {% include 'source/refresh_dhis2_credentials.html' %}
   <div class="form-nav">
-    <button class="btn btn-primary" type="submit" name="action" value="run_job">{{_("Refresh DHIS2 Harvester")}}</button>
+    <button class="btn btn-primary" type="submit" name="action" value="run_job">{{_("Refresh DHIS2 Source")}}</button>
```

Comment:
I wonder if this should this be `Refresh DHIS2 Import` rather than `Refresh DHIS2 Source`?